### PR TITLE
[Replenishment] Status of contributions excel export

### PR DESCRIPTION
**Pull request by andreighh**
- refs https://helpdesk.eaudeweb.ro/issues/25646

---

The full export is not very performant, but I think it's way more readable re-using the aggregators/worksheet functions. There might be some magic query similar to the Statistics sheet one (I couldn't find it :D), but I **think** the subqueries/sum filters + necessary python manipulations would bring it closer to these multiple queries. For a one-off export, it's still decent time.

### CODE REVIEW

File: core/api/export/replenishment.py [hunk 1]
```diff
@@ -26,10 +26,18 @@ def write_data(self, data):
             record_row.append(self.write_record_cell(cell_c))
 
             self.sheet.append(record_row)
+        self.sheet.merge_cells("A4:C4")
```
Comment (sorinstelian) [inline core/api/export/replenishment.py:29]: Please leave at least a comment stating why we need to do this. It's really not obvious. :)

File: core/api/views/utils.py [hunk 2]
```diff
@@ -428,3 +446,575 @@ def get_business_plan_from_request(request):
         raise ValidationError({"error": "Business plan not found"}) from e
 
     return business_plan
+
+
+class SummaryStatusOfContributionsAggregator:
```
Comment (sorinstelian) [inline core/api/views/utils.py:451]: This definitely deserves a docstring. ;)

**Review by sorinstelian — APPROVED** [review]
Looks good, but for the sake of future maintenance, please document the utility classes a bit.

File: core/api/export/replenishment.py (hunk 1 above)
Comment (andreighh) [reply core/api/export/replenishment.py:29]: Sorry, you're right.

**Review by sorinstelian — APPROVED** [review]
Looks good now! Great work!

**Pull request merged**